tests/README.md

```diff
@@ -5,6 +5,22 @@ This path contains the basic testing framework and integration testing cases, th
 ### Integration test
 The integration test is performed on every pull request and is run by the Drone CI. The script is stored in the `scripts/test-integration` file.
 
+Before the integration test, you need to check the apparmor status to ensure the `usr.sbin.virtqemud` is not enabled.
+Or you might see the following error when you test VM API
+```
+compute {"component":"virt-launcher","kind":"","level":"error","msg":"Failed to sync vmi","name":"test-grsvz","namespace":"default","pos":"server.go:202","reason":"virError(Code=1, Domain=10, Message='internal error: Failed to start QEMU binary /usr/bin/qemu-system-x86_64 for probing: libvirt:  error : cannot execute binary /usr/bin/qemu-system-x86_64: Operation not permitted\n')","timestamp":"2024-06-26T07:46:15.346012Z","uid":"e4ec1043-d9ce-4246-b672-f554e9a00169"}
+```
+
+You can use the following command to check:
+```
+$ aa-status |grep virtqemud
+```
+
+And use the following command to disable:
+```
+$ aa-disable /etc/apparmor.d/usr.sbin.virtqemud

```

Comment:
So we don't need to disable libvirtd as well ?

BTW I need to use sudo to make `aa-disable /etc/apparmor.d/usr.sbin.virtqemud` work.

Reply:
yeah, I tested, we only need to disable `virtqemud`.

I will add the sudo... thanks for the reminder!

Reply:
> yeah, I tested, we only need to disable virtqemud.
I see. Thanks for clarifying. 👍